cmd/layer/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	common "github.com/mimiro-io/common-datalayer"
+	pgl "github.com/mimiro-io/postgresql-datalayer/internal/layer"
+	"os"
+)
+
+func main() {
+	configFolderLocation := ""
+	args := os.Args[1:]
+	if len(args) >= 1 {
+		configFolderLocation = args[0]
+	}
+
+	pathFromEnv := os.Getenv("CONFIG_LOCATION")
```

Comment:
common datalayer already has DATALAYER_CONFIG_PATH built-in: https://github.com/mimiro-io/common-datalayer/blob/master/service.go#L30, I think it would be good to standardize on that across all layers

Reply:
agreed.